docs/README.md

```diff
@@ -1,80 +1,12 @@
 # Knowledge Base
 
-Below lies a list of resources that may be helpful to those looking to understand the go-control-plane API. The aim of these artifacts is to provide enough knowledge and understanding to newcomers and users who wish to use this API within their own codebases to implement an xDS compliant control-plane.
+Below lies a list of resources that may be helpful to those looking to understand the go-control-plane API.
 
-## Snapshot Cache
+## Implementations
 The following guides may be helpful on how to use go-control-plane's Snapshot Cache:
-- [Snapshot.md](cache/Snapshot.md)
-- [Server.md](cache/Server.md)
+- [cache.md](snapshot.md)
```

Comment:
nit: snapshot.md as link text?

Reply:
Ah oops thank you, it should be `cache.md`

Reply:
I think this was merged without updating this change, submitted https://github.com/envoyproxy/go-control-plane/pull/535